SPECS/bash/bash.spec

```diff
@@ -12,10 +12,10 @@ Source1:        bash_completion
 Patch0:         bash-5.1.patch
 BuildRequires:  readline
 Requires:       readline
-Requires(post): /bin/cp
-Requires(post): /bin/grep
-Requires(postun): /bin/grep
-Requires(postun): /bin/mv
+Requires(post): coreutils
```

Comment:
I'm fine with this change but file paths are used across many specs and in general should work, so we might need to think of a more future-proof solution, so that our package declare provides for directories, that are just symbolic links like `/bin` or `/lib`. An update to the steps ran after the `%install` section might do the trick. That's out of scope for this PR, though, but I was curious of your opinion. If you agree, I can add a bug for us to address this in the future.

Reply:
The Fedora packaging guide is fine with paths, but ONLY to `/usr/bin` (among some others).  I think the big issue here is we have a mixture of `/bin` and `/usr/bin`. Taking with someone on another team who ran into this a while ago `dnf` handles these mismatches properly, and either follows the symlinks or has some other logic to figure this out. `tdnf` however does not implement t his.